cdap-common/src/test/java/co/cask/cdap/common/zookeeper/coordination/ResourceCoordinatorTest.java

```diff
@@ -236,4 +245,22 @@ public InetSocketAddress getSocketAddress() {
       }
     };
   }
+
+  private Collection<PartitionReplica> waitForChange(BlockingQueue<Collection<PartitionReplica>> assignmentQueue,
```

Comment:
Ah yes, I should probably just add the timeout. Not sure what was I thinking.
Will update the PR